crates/containerd-shim-wasm/src/sandbox/instance.rs

```diff
@@ -32,16 +33,24 @@ where
     bundle: Option<String>,
     /// Namespace for containerd
     namespace: String,
+    // /// GRPC address back to main containerd
+    containerd_address: Option<String>,
 }
 
 impl<E> InstanceConfig<E>
 where
     E: Send + Sync + Clone,
 {
     pub fn new(engine: E, namespace: String) -> Self {
+        let os_args: Vec<_> = env::args_os().collect();
+        let containerd_address = match containerd::parse(&os_args[1..]) {
```

Comment:
I tried this in a few places (95d595541eeb47d67261c46223d837da6186093e / 89bb778251fd3f551849d350a11a79350712af5a) and landed here as a decent place but happy to hear feedback. 

I think really this shouldn't be something we need to do but should be passed in when the shim is created (https://github.com/containerd/runwasi/pull/147/files#diff-8f8704e2bb5644be97cfbd99c736cb7f432da2de20292de2b0f69e4193bd0f64R1360)

Reply:
fixed in #192